src/js/services/pull-request-storage-accessor.test.ts

```diff
@@ -81,7 +82,14 @@ describe('StorageSerialzer', () => {
       });
 
       it('loads empty arrays', () => {
-        expect(result).toEqual({ reviewRequested: [], noReviewRequested: [], allReviewsDone: [], missingAssignee: [], allAssigned: [] });
+        expect(result).toEqual({
+          reviewRequested: [],
+          teamReviewRequested: [],
+          noReviewRequested: [],
+          allReviewsDone: [],
+          missingAssignee: [],
+          allAssigned: [],
```

Comment:
Maybe rename to `myAssigned`?

Reply:
I'd rather not
<img width="253" alt="Screenshot 2021-11-23 at 08 48 00" src="https://user-images.githubusercontent.com/50821376/142987062-9ebea7fb-9738-461a-b6d7-66780c374ee8.png">